.github/workflows/release_begin.yaml

```diff
@@ -120,36 +129,28 @@ jobs:
             sha256="$(shasum --algorithm 256 ${{ github.workspace }}/crate_universe/private/bootstrap/bin/${triple}/release/${bin_name} | awk '{ print $1 }')"
             sed -i "s|{${triple}--sha256}|${sha256}|" ${{ github.workspace }}/crate_universe/private/defaults.bzl
           done
-
-          cat << EOF > ${{ github.workspace }}/tag_message.txt
-          From commit: ${GITHUB_SHA}
-
-          \`crate_universe_defaults.bzl\`:
-
-          \`\`\`python
-          $(cat ${{ github.workspace }}/crate_universe/private/defaults.bzl)
-          \`\`\`
-          EOF
         env:
-          # This url should match ${{ steps.crate_universe_release.outputs.html_url }} but because this step runs before
-          # `crate_universe_release` we hard code the url to be what we expect the release URL to be.
-          URL: https://github.com/bazelbuild/rules_rust/releases/download/crate_universe-${{ github.run_number }}
+          URL_PREFIX: https://github.com/${{ github.repository_owner }}/rules_rust/releases/download/${{ github.event.inputs.version }}
+      - run: |
+          # Update docs for release
+          SKIP_COMMIT=1 ${{ github.workspace }}/docs/update_docs.sh
```

Comment:
What's the effect of this, if we're not committing the output anywhere?

Reply:
This gets included in the PR that's created at the end of the file. I just wanted to run this script without committing so the PR only contained 1 commit. I almost didn't include this change because of https://github.com/bazelbuild/rules_rust/pull/695 which would make this not necessary but I did just to be safe.